src/sign/index.ts

```diff
@@ -55,6 +55,10 @@ export default class Client {
       'https://testnet.snapshot.org',
       constants.testnet.sequencer
     );
+    address = address.replace(
+      'https://localhost:3000',
```

Comment:
Maybe we can do like this, even better (on others env too)
```suggestion
      constants.local.hub,
```